spring-petclinic-api-gateway/src/main/java/org/springframework/samples/petclinic/api/ApiGatewayApplication.java

```diff
@@ -40,4 +46,31 @@ public static void main(String[] args) {
     RestTemplate loadBalancedRestTemplate() {
         return new RestTemplate();
     }
+
+    @Bean
+    public ZuulFilter zuulFilter(){
```

Comment:
Please use more descriptive names. What is a purpose of this filter? Isn’t it some kind of debug?

Reply:
No use reminant from lots of debugging to get this to work. Will take it out.